integration_testing/tools.go

```diff
@@ -41,18 +42,18 @@ const MaxSyncBlockHeight = 1500
 const HyperSyncSnapshotPeriod = 1000
 
 // get a random temporary directory.
-func getDirectory(t *testing.T) string {
+func getTestDirectory(t *testing.T, testName string) string {
 	require := require.New(t)
-	dbDir, err := ioutil.TempDir("", "badgerdb")
+	dbDir, err := ioutil.TempDir("", testName)
 	if err != nil {
 		require.NoError(err)
 	}
 	return dbDir
 }
 
-// generateConfig creates a default config for a node, with provided port, db directory, and number of max peers.
+// GenerateTestConfig creates a default config for a node, with provided port, db directory, and number of max peers.
 // It's usually the first step to starting a node.
-func generateConfig(t *testing.T, port uint32, dataDir string, maxPeers uint32) *cmd.Config {
+func GenerateTestConfig(t *testing.T, port uint32, dataDir string, maxPeers uint32) *cmd.Config {
```

Comment:
Any way we can deduplicate this and keep only a single `GenerateTestConfig` in cmd?

Reply:
SGTM. I'll do this change